src/ExpandableContainer/ExpandableContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import * as FontAwesome from "react-fontawesome";
+import * as classnames from "classnames";
+import * as PropTypes from "prop-types";
+
+import FlexBox from "../flex/FlexBox";
+import FlexItem from "../flex/FlexItem";
+
+import "./ExpandableContainer.less";
+
+export interface Props {
+  children: React.ReactNode;
+  className?: string;
+  isExpanded?: boolean;
+  onClick?: () => void;
+  title?: React.ReactNode;
```

Comment:
If the title is optional, it's possible to make an inaccessible version of this, I think, where there's no label, no aria-label, but there _is_ click functionality.

I suggest adding an `ariaLabel?: string` prop, which is stuck onto the FlexBox that has the onClick below; that way, consumers can make sure themselves that it's accessible and uniquely labeled. Making a note of this in the component's prop descriptions is also probably a best practice we should adopt. 😄

Reply:
wow, good catch, you're an expert on accessibility! I'm more than happy to do that, but this also got me thinking, maybe title should be required, since I don't imagine this expandable container being too useful without the title. What do you think?

Reply:
Yeah, I think that's actually the better route to go! I, too, can't see much of a use case for no title, and now the consumer can't accidentally create an inaccessible instance of the component. 🙂